flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNodeConfig.java

```diff
@@ -87,18 +86,18 @@ public long getStateRetentionTime() {
 
     // See https://issues.apache.org/jira/browse/FLINK-26190
     /**
-     * Using {@link #plannerConfig} to satisify tests like {@code OverAggregateHarnessTest}, which
+     * Using {@link #tableConfig} to satisify tests like {@code OverAggregateHarnessTest}, which
      * use {@code HarnessTestBase#TestTableConfig} to individually manipulate the
```

Comment:
This comment is outdated. In theory we could also move this method to `TableConfigUtils`, right? This would keep `ExecNodeConfig` clean.

Reply:
Done, thx!